src/components/solutions/task07/NewTodoForm.js

```diff
@@ -0,0 +1,31 @@
+import { React, useState } from "react";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
It is better avoid using external libraries and find a work around (random id with math function)